tests/classdefs.pyi

```diff
@@ -1,4 +1,4 @@
-# flags: --extend-ignore=Y023
+# flags: --extend-ignore=F821,Y023
```

Comment:
should we just add `F821` to our `per-file-ignore` for `.pyi` files here rather than adding this flag to multiple test files?

https://github.com/PyCQA/flake8-pyi/blob/541330a3a9b4c777a0e061d967b0ce780c4050be/.flake8#L33